WordPress/Classes/Utility/ZendeskUtils.swift

```diff
@@ -361,6 +372,7 @@ private extension ZendeskUtils {
         ZDKConfig.instance().userIdentity = zendeskIdentity
         DDLogDebug("Zendesk identity created with email '\(zendeskIdentity.email)' and name '\(zendeskIdentity.name)'.")
```

Comment:
I guess unrelated to this PR, but I noticed that these values aren't unwrapped before they're logged, resulting in:

`2018-05-31 12:09:29:447 WordPress[93831:35215147] Zendesk identity created with email 'Optional("demo@example.com")' and name 'Optional("My Name")'.`